frontend/src/app/components/enterprise-banner/enterprise-banner.component.sass

```diff
@@ -1,4 +1,4 @@
-.notification-box
+.op-enterprise-banner-notification-box
```

Comment:
The name of the block should be the name of the sass file

Reply:
So, like in the other comment on this, I'd recommend renaming this to `enterprise-banner`, or renamign the file.